docs/management/managing-indices.asciidoc

```diff
@@ -26,8 +31,9 @@ See {ref}/security-privileges.html[Security privileges] for more
 information.
 
 Before using this feature, you should be familiar with index management
```

Comment:
PR #80486 is adding some content to this section.  Maybe we should consider creating a "Prequisites" section with the suggested reading and the required permissions.